File: java/v1/src/main/java/com/api/v1/book/helpers/IsbnGeneratorUtil.java
package com.api.v1.book.helpers;

import java.security.SecureRandom;

public interface IsbnGeneratorUtil {

    static String generateIsbn() {
        long min = 1000000000000L;
        long max = 9999999999999L;
        long isbn = new SecureRandom().nextLong(min, max+1);
        return String.valueOf(isbn);
    }

}

File: koltlin/v1/src/main/kotlin/com/api/v1/book/utils/IsbnGeneratorUtil.kt
package com.api.v1.book.utils

import java.security.SecureRandom

class IsbnGeneratorUtil {

    companion object {
        fun generator(): String {
            val min: Long = 1000000000000L
            val max: Long = 9999999999999L
            return SecureRandom().nextLong(min, max+1).toString()
        }
    }

}
